### DISCUSSION TITLE
Next Release Version 6

### DISCUSSION THREAD

**Discussion by Steffenberg** [thudugala/Plugin.LocalNotification#151] [Q&A] [answered]
Hello there,

quick question: when do you plan to release the next version?

Best regards

**Comment by thudugala** [comment]
@Steffenberg in the process of merging #147. 

The PR is suggesting that we should use NotificationCenter without using NotificationCenter.Current. So it it align with Xamarin Essential plugin design pattern. 

But then we lose feasibility of INotificationService for MVVM for design pattern and developerd have to create there own interfaces.

**Reply by ArunPrakashG** [reply]
Any updates regarding this ?

**Reply by thudugala** [reply]
@ArunPrakashG What is your view on drooping NotificationCenter.Current.

**Reply by ArunPrakashG** [reply]
Not exactly a positive view regarding that. mainly due to it being a Breaking change.

**Reply by Steffenberg** [reply]
Sorry for the late reply. I personally would also not like dropping the NotificationCenter.Current that much just get near to the Xam.Essentials plugin design patterns. But I guess this is more a question of faith or taste.

**Reply by ArunPrakashG** [reply]
@thudugala any update on when the next release is going to be ?

**Comment by thudugala** [comment, marked as answer]
From version 10.0.0 in .Net MAUI, you can use INotificationService. it is dependency injected.
